ansible/roles/web/templates/parameters.yml.j2

```diff
@@ -45,3 +45,7 @@ parameters:
     # Register at https://sentry.io
     sentry_dsn:        '{{ site_sentry_dsn }}'
     sentry_env:        '{{ site_sentry_env }}'
+
+    # Algolia Configuration
+    env(ALGOLIA_APP_ID):  '{{ site_algolia_app_id }}'
```

Comment:
This part should stay, disregard my comment in #246 
Algolia client v3 has changed the configuration interface.